scripts/sct_extract_metric.py

```diff
@@ -112,6 +112,11 @@ def get_parser():
                       mandatory=False,
                       example='2:5',
                       default_value=param_default.vertebral_levels)
+    parser.add_option(name='-vertfile',
+                      type_value='str',  # note: even though it's a file, we cannot put the type='file' otherwise the full path will be added in sct_testing and it will crash
```

Comment:
I'm not sure this is the right solution to the problem. We lose error checking in the process.

Reply:
as mentioned in my comment, if we put "file", the full path will be added to sct_testing and then it will crash because it does not find the file. I know this is a current fundamental problem in the way sct_testing works, but that would require several hours to fix that. I don't have those few hours for the next few weeks, and having -vertfile is more important than having the file check. So this is just a question of priority. We can open an issue to address the file checking.